packages/loader/container-loader/src/container.ts

```diff
@@ -2456,16 +2460,27 @@ export class Container
 			() => this.connected,
 			this._deltaManager.clientDetails,
 			existing,
+			LoaderCompatDetails,
```

Comment:
nit. this could just be inlined in the context rather than passes here, used as a private memeber, than exposed via a getter